core-web/libs/portlets/dot-ema/src/lib/feature/store/dot-ema.store.ts

```diff
@@ -37,7 +39,22 @@ export class EditEmaStore extends ComponentStore<EditEmaState> {
     readonly language_id$: Observable<string> = this.select((state) => state.language_id);
     readonly url$: Observable<string> = this.select((state) => state.url);
     readonly pageTitle$: Observable<string> = this.select((state) => state.editor.page.title);
+    readonly editIframeURL$: Observable<string> = this.select((state) => state.editIframeURL);
 
+    readonly vm$ = this.select((state) => {
+        return {
+            iframeUrl: `http://localhost:3000/${state.url}?language_id=${state.language_id}`,
+            language_id: state.language_id,
+            pageTitle: state.editor.page.title,
+            editIframeURL: state.editIframeURL
```

Comment:
```suggestion
            iframeURL: state.editIframeURL
```

Right now is edit but soon will be `add`, `select`, etc...

Reply:
Changed!